src/ServiceControlInstaller.Engine/Instances/Transports.cs

```diff
@@ -17,37 +17,44 @@ public static TransportInfo FindByName(string name)
             {
                 Name = "AzureServiceBus",
                 TypeName = "NServiceBus.AzureServiceBusTransport, NServiceBus.Azure.Transports.WindowsAzureServiceBus",
-                MatchOn = "NServiceBus.AzureServiceBus",
+                MatchOn = "NServiceBus.AzureServiceBusTransport",
```

Comment:
@johnsimons Changing the match value will break upgrades from older ServiceControl that are still on NSB v4.  The V4 transports didn't have "Transport" at the end of the string.